views/widget.subscribe.display.php

```diff
@@ -5,7 +5,10 @@
     * values as needed.
     */
     $widget_id = $this->id;
+   
     $title = empty( $instance['title'] ) ? ' ' : apply_filters( 'widget_title', $instance['title'] );
+    $source = empty( $instance['source'] ) ? bloginfo( 'url' )  :  $instance['source'];
```

Comment:
this works, though is a little confusing. I usually go with the PHP's shorthand ternary:

```php
$source = $instance['source'] ?: bloginfo('url');
```

Will use bloginfo if instance['source'] is empty.